transformations/style_paraphraser/test.json

```diff
@@ -0,0 +1,59 @@
+{
+    "type": "style_paraphraser",
```

Comment:
Can you add a test for very long input to ensure correct behavior?

Reply:
Fixed! Also made it able to better handle longer input by paraphrasing sentence by sentence.